build.gradle.kts

```diff
@@ -253,6 +282,7 @@ tasks {
             extraArgs.add("-Dlincheck.traceDebuggerMode=true")
             exclude("**/lincheck_test/guide/*")
         }
+        // TODO: do we need testInTraceRecorderMode in the gradle.properties then?
```

Comment:
Don't think we need it. The flag was mostly used to selectively disable some Lincheck tests that are not yet supported in trace debugger mode. Since trace recorder does not re-use `ManagedStrategy`, there should be no overlap between the tests.